target_chains/solana/pyth_solana_receiver_sdk/src/price_update.rs

```diff
@@ -59,29 +59,30 @@ pub struct PriceUpdateV2 {
 impl PriceUpdateV2 {
     pub const LEN: usize = 8 + 32 + 2 + 32 + 8 + 8 + 4 + 8 + 8 + 8 + 8 + 8;
 }
-/// A time weighted average price account. This account is used by the Pyth Receiver program to store a verified TWAP update from a Pyth price feed.
+/// A time weighted average price account.
+/// This account is used by the Pyth Receiver program to store a TWAP update from a Pyth price feed.
+///
+/// ## Warning
+/// This function doesn't check whether the TwapUpdate is constructed from verified price updates.
+///     TwapUpdates should be only created and used after the client has already verified the VAAs via the Wormhole contract.
+///     Check out `target_chains/solana/cli/src/main.rs` for an example of how to do this.
```

Comment:
What do you mean here? From what i understand the TwapUpdate can be created only from verified VAAs (we get VAA accounts that are owned by wormhole and we check the sources)

Reply:
yeah your understanding is correct, the comment is a bit confusing -- will update